docs/DeveloperGuide.adoc

```diff
@@ -354,31 +364,52 @@ image::UndoRedoActivityDiagram.png[width="650"]
 ** Pros: Easy to understand for new Computer Science student undergraduates to understand, who are likely to be * the new incoming developers of our project.
 ** Cons: Logic is duplicated twice. For example, when a new command is executed, we must remember to update * both `HistoryManager` and `UndoRedoStack`.
 * **Alternative 2:** Use `HistoryManager` for undo/redo
-** Pros: We do not need to maintain a separate stack, and just reuse what is already in the codebase.
-** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two * different things.
+** Pros: We do not need to maintain a separate stack, and can just reuse what is already in the codebase.
+** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Furthermore, the Single Responsibility Principle and Separation of Concerns are violated as `HistoryManager` now needs to do two * different things.
```

Comment:
Change to "Separation of Concerns and Single Responsibility principles"

Reply:
they're 2 different principles. Or did you mean "Separation of Concerns" and "Single Responsibility" principles ?

Reply:
Yeah, "Separation of Concerns" and "Single Responsibility" principles.
Otherwise, "the Single Responsibility Principle and Separation of Concerns Principle are violated". Though, I feel the former sounds better

Reply:
👍